sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -1604,3 +1605,6 @@ def test_piecewise__eval_is_meromorphic():
     assert f.is_meromorphic(x, 2) == True
     assert f.is_meromorphic(x, Symbol('a')) == None
     assert f.is_meromorphic(x, Symbol('a', real=True)) == None
+
+def test_25603():
+    assert parse_expr("Piecewise((log(x), (x <= 5) & (x > 3)), (x, True))").simplify() == Piecewise((log(x), (x <= 5) & (x > 3)), (x, True))
```

Comment:
We don't need to use `parse_expr` here.

Reply:
ok lemme change that up

Reply:
I think there is still a conditional case needed to be added but I am not sure. Pls check once